services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/DigitalGreenCertificateToCborMapping.java

```diff
@@ -49,7 +49,12 @@ public DigitalGreenCertificateToCborMapping(DigitalCovidCertificateClient digita
    */
   public List<BusinessRule> constructRules(RuleType ruleType)
       throws DigitalCovidCertificateException, FetchBusinessRulesException {
-    List<BusinessRuleItem> businessRulesItems = digitalCovidCertificateClient.getRules();
+    List<BusinessRuleItem> businessRulesItems;
+    if (ruleType.name().equalsIgnoreCase(RuleType.BoosterNotification.name())) {
```

Comment:
Maybe we can use the enum to store the corresponding logic behind this in order to replace the if statement? For me the if-statement feels wrong here

Reply:
We tried the enum approach and did not work.